src/components/Button/index.module.css

```diff
@@ -0,0 +1,29 @@
+@import "../../styles/settings.css";
+
+.root {
+  display: inline-block;
+
+  padding: $size-26 $size-36;
+  border-radius: $size-26;
+
+  border: none;
+
+  font-size: $size-26;
+  line-height: $size-36;
+  font-family: "Gil Sans";
```

Comment:
You can cut this off, we set the `font-family` globally elsewhere

Reply:
Done 👍